README.md

```diff
@@ -1,3 +1,11 @@
 # Specs
 
 This repo maintains rfcs and definitions for Beyond Storage.
+
+## RFC Process
+- (Optional) Submit an issue in `beyondstorage/specs` or [forum](https://forum.beyondstorage.io/) as a pre-proposal to discuss the idea.
+- Copy `rfcs/0-example.md` to `rfcs/0-my-feature.md` (where "my-feature" is descriptive). Don't assign an RFC number yet; This is going to be the PR number.
+- Fill in the RFC.
+- Submit a pull request. Use the issue number of the PR to update the filename.
```

Comment:
If the proposal is huge or complex, the reviewer could require the author to implement a demo.